Automattic/woocommerce-payments
# HACK week: migration to TypeScripts (client/payment-details)

This PR partially migrates `client/payment-details` folder components onto TypeScript.

Testing instructions:
- all CI checks related to JS are passing
- open a transaction details and ensure the summary block on top of the page is visually the same as for develop branch
